src/amo/pages/Home/index.js

```diff
@@ -211,7 +214,17 @@ export class HomeBase extends React.Component {
       <ul className="Home-SubjectShelf-list">
         {curatedThemes.map(({ color, slug, title }) => (
           <li className="Home-SubjectShelf-list-item" key={slug}>
-            <Link to={`/themes/${slug}/`} className="Home-SubjectShelf-link">
+            <Link
+              to={{
```

Comment:
It would be helpful to make a function that returns the `to` object and share it with the other one:

https://github.com/mozilla/addons-frontend/blob/48dac4d7e560654f7ae93cdd5acc53a68ed960af/src/amo/components/Categories/index.js#L151-L158

That will make it easier to maintain any future changes to the `sort` value or something like that.

Reply:
Good idea. This helps with the tests too.